python/ray/tune/trainable.py

```diff
@@ -153,6 +154,10 @@ def get_current_ip(self):
         self._local_ip = ray.util.get_node_ip_address()
         return self._local_ip
 
+    def is_actor(self):
+        actor_id = ray.worker.global_worker.actor_id
```

Comment:
can you try catch this just in case? i think ray.worker.global_worker might not be instantiated

Reply:
Done!